geth/chain.py

```diff
@@ -89,18 +90,16 @@ def is_ropsten_chain(data_dir):
 def write_genesis_file(
     genesis_file_path,
     overwrite=False,
-    nonce="0xdeadbeefdeadbeef",
+    nonce="0x0",
```

Comment:
Just curious, where do the updates to genesis fields from?

Reply:
Removing clique since it's no longer supported. Set others to more appropriate values but can put them back. These are just default values updates. `Difficulty == 0` bc post-merge, etc.